string link_name          # name of link
string reference_frame    # reference frame of returned information, must be a valid link
                          # if empty, use inertial (gazebo world) frame
---
gazebo/LinkState link_state
bool success              # return true if get info is successful
string status_message     # comments if available
